### PR TITLE
Fix: improve behaviour of `selection` in `df.viz.histogram`

### PR DESCRIPTION
**Pull request by JovanVeljanoski**
Addresses: https://github.com/vaexio/vaex/issues/2142

### CODE REVIEW

File: packages/vaex-viz/vaex/viz/mpl.py [hunk 1]
```diff
@@ -140,6 +142,8 @@ def histogram(self, x=None, what="count(*)", grid=None, shape=64, facet=None, li
                     raise ValueError("Could not understand 'what' argument %r, expected something in form: 'count(*)', 'mean(x)'" % what)
         else:
             grid = self.histogram(binby, size=shape, limits=limits, selection=selection)
+    if (len(grid.shape) > 1) & (_issequence(selection)):
```
Comment (maartenbreddels) [inline packages/vaex-viz/vaex/viz/mpl.py:145]: And
```suggestion
    if (len(grid.shape) > 1) and (_issequence(selection)):
```

File: packages/vaex-viz/vaex/viz/mpl.py [hunk 2]
```diff
@@ -1,18 +1,19 @@
 import ast
 import functools
+from pydoc import doc
```
Comment (maartenbreddels) [inline packages/vaex-viz/vaex/viz/mpl.py:3]:
```suggestion
```

**Review by maartenbreddels — CHANGES_REQUESTED** [review]
Some minor things

**Comment by JovanVeljanoski** [comment]
@maartenbreddels approved telepathically (and the failures are unrelated)

**Pull request merged**